flict/flictlib/project/reader.py

```diff
@@ -48,71 +48,95 @@ def _add_files_read(self, project_file):
     def _already_read(self, project_file):
         return project_file in self.files_read
 
+    def prepare_project(self, project):
+        for package in project['packages']:
```

Comment:
this assumes that `packages` is a mandatory item in project - I think it's better to use `project.get('package', [])`